src/fu-util-common.c

```diff
@@ -2350,12 +2351,15 @@ fu_util_security_attrs_to_string(GPtrArray *attrs, FuSecurityAttrToStringFlags s
 			    !fwupd_security_attr_has_flag(attr, FWUPD_SECURITY_ATTR_FLAG_SUCCESS))
 				low_help = TRUE;
 
-			/* check for PCR0 not matching */
-			if (g_strcmp0(fwupd_security_attr_get_appstream_id(attr),
-				      FWUPD_SECURITY_ATTR_ID_TPM_RECONSTRUCTION_PCR0) == 0 &&
-			    fwupd_security_attr_get_result(attr) ==
-				FWUPD_SECURITY_ATTR_RESULT_NOT_VALID)
-				pcr0_help = TRUE;
+			if (fwupd_security_attr_get_result(attr) ==
+			    FWUPD_SECURITY_ATTR_RESULT_NOT_VALID) {
```

Comment:
it almost feels like we need a result of `MISSING_DATA` that plugins can emit -- that results in the HSI result being invalid because some required data is not present.

Reply:
If we go with the `MISSING_DATA` idea we can also add in a plugin for other unsupported architectures to just emit missing data too.
This makes me tempted then to make `fwupdmgr security` not require --force unless no `MISSING_DATA`.

Reply:
Can you split this bit out as a new PR and branch and I'll have a play next week.

Reply:
https://github.com/fwupd/fwupd/pull/4481